.devcontainer/devcontainer.json

```diff
@@ -17,7 +16,8 @@
       ]
     }
   },
-  "updateContentCommand": "[ -f packages.txt ] && sudo apt update && sudo apt upgrade -y && sudo xargs apt install -y <packages.txt; [ -f requirements.txt ] && pip3 install --user -r requirements.txt; pip3 install --user streamlit; echo '✅ Packages installed and Requirements met'",
+  "updateContentCommand": "pip install --upgrade pip && pip install -r trends/requirements.txt",
```

Comment:
The container didn't work at first when I started it, so I asked cursor to fix it based on the error and this is what it did. To be honest, I hadn't previously used devcontainers, so I'm not entirely sure if this is a better way to do it or not, but it did work for me. Happy to roll this back as well.